coding-standards/php.md

```diff
@@ -11,5 +11,317 @@ https://www.php-fig.org/psr/psr-12/
 [https://cheatsheetseries.owasp.org/cheatsheets/Laravel_Cheat_Sheet.html](https://cheatsheetseries.owasp.org/cheatsheets/PHP_Configuration_Cheat_Sheet.html)
```

Comment:
Laravel do we need still? We decided to have it for basic PHP code first.
If they are common for basic PHP also then fine.

Reply:
Can we have similar title as that of dotnet?

https://github.com/sai-manohar/dev-standards/blob/main/coding-standards/dotnet.md

- C# coding standards
- .NET security best practices

We can decide a common term later on but for now just replace tech infront and keep the same heading

Reply:
Followed the headings similar to dot net. 
Updated security doc to generic PHP instead of laravel.